packages/yuudachi/src/util/actionKeys.ts

```diff
@@ -1,2 +1,47 @@
-export const ACTION_KEYS = ["restriction", "", "warn", "kick", "softban", "ban", "unban", "timeout", ""] as const;
-export const REPORT_KEYS = ["pending", "approved", "rejected", "spam"] as const;
+import i18next from "i18next";
+import { CaseAction } from "../functions/cases/createCase.js";
+import { ReportStatus } from "../functions/reports/createReport.js";
+
+export function caseActionLabel(key: CaseAction, locale: string, isCase = false): string {
+	switch (key) {
+		case CaseAction.Role:
+			return isCase
+				? i18next.t(`log.history.cases.action_label.restriction_case`, { lng: locale })
+				: i18next.t(`log.history.cases.action_label.restriction`, { lng: locale });
+		case CaseAction.Unrole:
+			return isCase
+				? i18next.t(`log.history.cases.action_label.unrestriction_case`, { lng: locale })
+				: i18next.t(`log.history.cases.action_label.unrestriction`, { lng: locale });
```

Comment:
Whether you decide to inline the ternary or not, you should at least use `"` and not backticks then, if it stays like it is now.

Reply:
Ow, true, didn't realize